### DISCUSSION TITLE
Queda espacio en blanco en las notas

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#481] [4. Bug Reports]
Hola a todos

Me di cuenta que en Notas, se encuentran separadas, por un espacio vacío. No sé si es por un tema de ordenamiento por semana / mes o puede ser un bug.

Dejo la imagen

Gracias =>
![image](https://user-images.githubusercontent.com/61419471/137749300-1b554f07-138d-41e1-925d-4ef2e8d1baeb.png)

**Comment by jdurandpaz** [comment]
se crea issue https://github.com/Cotalker/documentation/issues/486
